### ISSUE TITLE
Translating submissions & application forms

### ISSUE THREAD

**Issue by wes-otf** [HyphaApp/hypha#3814]
OTF has recently received applications fully in Spanish, Russian, and Arabic. An interesting feature to explore would be integrated translations via an open source engine like [Argos Translate](https://github.com/argosopentech/argos-translate). As the adopters of Hypha continue to span across different languages I think this would be super useful in terms of being able to translate applications forms & submissions.

Would love to hear thoughts about this!

**Comment by frjo** [comment]
Technically I do not think it is that hard to get it up and running.

The hard part is to create a UX for it.

There is also the issue with machine translating quality. They tend to be ok between Indo-European languages. To and from other language groups the results are … mixed.

**Comment by wes-otf** [comment]
That makes sense. I played a bit with the mentioned Argos Translate and did a test translation of an application that we received that was in Arabic. We have a team member that is fluent, so I passed the translations off to them to give some feedback on. If this at least provides the ability to get the gist (without misrepresenting the submission) and make an initial determination, I would still see that as win to keep info contained within the app (especially if indo-european langs had a high accuracy rate).

Lots more testing needed but if this was something we decide explore, I could see it being a beta feature for a while (probably only to staff) and giving some prefaces on accuracy of translations before use. Will update with the feedback I get.
